src/apps/datasets/models.py

```diff
@@ -88,7 +88,7 @@ def in_use(self):
         return competitions_in_use
 
     def __str__(self):
-        return self.name or ''
+        return f'{self.name}({self.id})' or ''
```

Comment:
```suggestion
        return f'{self.name}({self.id})'
```
I don't believe the 2nd operand of that `or` will ever be executed.